docs/supported-options-v0.2.x.md

```diff
@@ -0,0 +1,46 @@
+#### Supported options for v0.2.x
+
+Flag | Env variable | Default value | Description
+--- | --- | --- | ---
+`--otc-access-key-id`     | `ACCESS_KEY_ID`        |                                     | Access key ID for AK/SK auth
+`--otc-access-key-key`    | `ACCESS_KEY_SECRET`    |                                     | Secret access key for AK/SK auth
+`--otc-auth-url`          | `OS_AUTH_URL`          | https://iam.eu-de.otc.t-systems.com | Authentication URL
+`--otc-availability-zone` | `OS_AVAILABILITY_ZONE` | eu-de-03                            | Availability zone
+`--otc-available-zone`    | `AVAILABLE_ZONE`       |                                     | Availability zone. **DEPRECATED**: use `-otc-availability-zone` instead
+`--otc-bandwidth-size`    | `BANDWIDTH_SIZE`       | 100 (MBit/s)                        | Bandwidth size
+`--otc-bandwidth-type`    | `BANDWIDTH_TYPE`       | PER (exclusive bandwidth)           | Bandwidth share type
+`--otc-cloud`             | `OS_CLOUD`             |                                     | Name of cloud in `clouds.yaml` file
+`--otc-domain-id`         | `OS_DOMAIN_ID`         |                                     | OpenTelekomCloud Domain ID
+`--otc-domain-name`       | `OS_DOMAIN_NAME`       |                                     | OpenTelekomCloud Domain name
+`--otc-elastic-ip`        | `ELASTIC_IP`           | 1                                   | If set to 0, elastic IP won't be created. **DEPRECATED**: use `-otc-skip-ip` instead
+`--otc-elastic-ip-type`   | `ELASTICIP_TYPE`       |                                     | Bandwidth type. **DEPRECATED!** Use `-otc-floating-ip-type` instead
```

Comment:
ELASTICIP_TYPE is ok, or should be ELASTIC_IP_TYPE?

Reply:
This is the old flag documentation

Reply:
yes, it's wrong, but in version `v0.2.4` we already had this typo

```
		mcnflag.StringFlag{
			Name:   "otc-elastic-ip-type",
			EnvVar: "ELASTICIP_TYPE",
			Usage:  "OpenTelekomCloud bandwidth type. DEPRECATED! Use -otc-floating-ip-type instead",
		},
```